docs/website/docs/installation.mdx

```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
 
 ## Set up environment
 
-## Install `dlt` library
\ No newline at end of file
+Make sure you are using **Python 3.8 or 3.9**:
+
+```bash
+python3 --version
+pip3 --version
+```
+
+If not, then please follow the instructions below to install it:
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu">
+
+You can install Python 3.9 with an `apt` command.
```

Comment:
we should go for python 3.10 btw. this is actually version on which we test on github